Teams/30 Create Embedded Web Experiences with Tabs for Microsoft Teams/Demos/03-auth-tab/src/app/web/auth-end.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 <html>
+
 <body>
-  <script src="https://statics.teams.cdn.office.net/sdk/v1.5.2/js/MicrosoftTeams.min.js" crossorigin="anonymous"></script>
+  <script src="https://statics.teams.cdn.office.net/sdk/v1.5.2/js/MicrosoftTeams.min.js"
+    crossorigin="anonymous"></script>
   <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal.min.js" crossorigin="anonymous"></script>
 
   <script type="text/javascript">
     microsoftTeams.initialize();
 
     // ADAL.js configuration
     let config = {
-      clientId: "fa792ad0-1f96-4807-b652-929e8efbaf95",
+      clientId: "da0770c2-f800-4ab4-870e-8ffae5d39f6c",
```

Comment:
Hmm. Are we sure we want to hardcode this into the repo? Maybe we want to add a placeholder like {----replace with your clientId----}? Unless they are not expected to change it and will use this clientId instead for the purposes of training.